dev_ops/sql/Tables/CreateUsers.sql

```diff
@@ -0,0 +1,10 @@
+CREATE TABLE users (
+    id BIGSERIAL PRIMARY KEY,
+	twitch_username TEXT UNIQUE,
+	mixer_username TEXT UNIQUE,
+	youtube_username TEXT UNIQUE,
```

Comment:
Same text comment here. Sine we dont intend to have mixer/youtube for awhile, do you think we should leave them off for now and run an UPDATE script when we're ready for them?

Reply:
Also, what about this scenario. A user logs in with twitch. Then they log out and then login with mixer. How do we know those accounts are the same? I actually think only 1 of these is going to be define at once, the rest will be NULL.

Reply:
Will switch these to VARCHAR(128).

I do think it's valid to have all three for a single account. You can imagine a user logging with twitch, but then wishing to also accept challenges on Mixer. We could have a feature like "link your Mixer account" that would then update this field.

I don't see much harm in adding these fields, even if we don't use them right away. Changing schemas down the road though could be much more difficult.

Reply:
I think predicting use cases is tough to get right. I dont see a use case for these now and as such i think we should keep them off and run an update script down the road if they are needed. But im fine leaving them if you feel strongly.